lib/teamcityBrowser.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+/**
+ * Teamcity doc reference https://confluence.jetbrains.com/display/TCD10/Build+Script+Interaction+with+TeamCity
+ *
+ * Module dependencies.
+ */
+var Base = window.Mocha.reporters.Base;
+var log;
+if(typeof window.customLogFunction === 'function'){
+	log = window.customLogFunction;
```

Comment:
I would prefer this to be in the Mocha reporter options instead of a global, since that's the standard way to do it.